### PR TITLE
boards/soc: Rename to use vendor prefixes and misc. fixes

### PR DESCRIPTION
**Pull request by nordicjm**
Some changes that were missed in the collab-hwm branch.

Compliance failure is expected due to large images being moved into different folder names

### CODE REVIEW

**Review by tejlmand — COMMENTED** [review]
Only concern is that `up` as vendor prefix seems to be a little too generic.

File: dts/bindings/vendor-prefixes.txt [hunk 1]
```diff
@@ -678,6 +682,7 @@ udoo	Udoo
 ugoos	Ugoos Industrial Co., Ltd.
 ultrachip	UltraChip Inc.
 uniwest	United Western Technologies Corp (UniWest)
+up	UP Bridge the Gap
```
[the remark this answers is not in the document]
Comment (nordicjm) [reply dts/bindings/vendor-prefixes.txt]: Changed to `up-bridge-the-gap`

**Review by tejlmand — DISMISSED** [review]

File: dts/bindings/vendor-prefixes.txt [hunk 2]
```diff
@@ -72,6 +73,7 @@ arm	ARM Ltd.
 armadeus	ARMadeus Systems SARL
 arrow	Arrow Electronics
 artesyn	Artesyn Embedded Technologies Inc.
+arturo182	Arturo182
```
Comment (henrikbrixandersen) [inline dts/bindings/vendor-prefixes.txt:76]: This vendor should like be "solderparty" or similar, arturo182 is just his username. Another option would be to put them under "others".

https://www.tindie.com/stores/arturo182/

**Review by henrikbrixandersen — DISMISSED** [review]
Awesome. One minor suggestion.

**Review by nashif — DISMISSED** [review]

**Review by tejlmand — DISMISSED** [review]

File: dts/bindings/vendor-prefixes.txt (hunk 2 above)
Comment (nordicjm) [reply dts/bindings/vendor-prefixes.txt]: Fixed

**Review by henrikbrixandersen — DISMISSED** [review]

**Review by decsny — DISMISSED** [review]

**Review by tejlmand — DISMISSED** [review]

**Review by nashif — DISMISSED** [review]

**Review by carlescufi — DISMISSED** [review]

**Comment by carlescufi** [comment]
Compliance failure is about existing image size.

**Review by fabiobaltieri — DISMISSED** [review]

**Review by tejlmand — DISMISSED** [review]

**Comment by henrikbrixandersen** [comment]
@nordicjm Merge conflict in `boards/synopsys/nsim/board.cmake`.

**Review by carlescufi — APPROVED** [review]

**Review by henrikbrixandersen — APPROVED** [review]

**Review by fabiobaltieri — APPROVED** [review]

**Review by ruuddw — APPROVED** [review]

**Pull request merged**